src/clients/authorization/AuthorizationEvaluate.tsx

```
@@ -296,11 +328,23 @@ export const AuthorizationEvaluate = ({
               helperTextInvalid={t("common:required")}
               fieldId={name!}
             >
-              <AttributeInput
-                selectableValues={resources.map((item) => item.name!)}
-                resources={resources}
-                isKeySelectable
-                name="resources"
+              <Controller
+                name="applyToResource"
+                defaultValue=""
+                control={control}
+                render={() => (
+                  <AttributeInput
+                    selectableValues={
+                      resources
+                        .map((item) => [{ name: item.name }])
+                        .flat() as AttributeType[]
```

There is no need to call `flat()` here or to type-cast the items if you map it directly to an object instead of an array with a generic type.

```ts
resources.map<AttributeType>((item) => ({
  name: item.name!,
}))
```

According to the `AttributeType` type you will also have to provide a `key` which is missing here. Because this was being type-casted this error was suppressed by the compiler.